### PR TITLE
Complete assigned features

### PR DESCRIPTION
**Pull request by YANLING-H**
1. refactor the interview information in the interviewDetailCard and interviewCard, name it as interviewDetailHeader
2.  show only one note under the interview detail in the [note list screen](https://www.figma.com/file/SptJLDZmGfgT1NJruX88PK/InterPrep?type=design&node-id=350-4487&mode=design&t=FLTvXiijJgcperZg-4) and [view note list screen](https://www.figma.com/file/SptJLDZmGfgT1NJruX88PK/InterPrep?type=design&node-id=171-4588&mode=design&t=FLTvXiijJgcperZg-4)

### CODE REVIEW

File: app/src/main/kotlin/com/twain/interprep/presentation/ui/components/note/InterviewDetailForNote.kt [hunk 1]
```diff
@@ -72,6 +63,35 @@ fun InterviewDetailForNote(
...
 
+@Composable
+fun InterviewDetails(
+    height: Dp,
+    interview: Interview,
+    companyTextColor: Color,
+    companyTextStyle: TextStyle = MaterialTheme.typography.bodyMedium,
+    roundTypeTextColor: Color,
+    roundTypeTextStyle: TextStyle = MaterialTheme.typography.bodyMedium) {
+    Box(modifier = Modifier.height(height)) {
```
Comment (YANLING-H) [inline app/src/main/kotlin/com/twain/interprep/presentation/ui/components/note/InterviewDetailForNote.kt:74]: Do we need to fix the height of the box?

File: app/src/main/kotlin/com/twain/interprep/presentation/ui/components/note/NoteCard.kt [hunk 2]
```diff
@@ -65,44 +65,42 @@ fun NoteCard(
...
-                        Column(
-                            verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.dimension_4dp))
-                        ) {
-                            Text(
-                                text = note.interviewSegment,
-                                style = MaterialTheme.typography.bodyLarge
-                            )
+                        Text(
+                            text = 0.toString(),
+                            style = MaterialTheme.typography.titleMedium
```
Comment (YANLING-H) [inline app/src/main/kotlin/com/twain/interprep/presentation/ui/components/note/NoteCard.kt:84]: Do we need to keep the index in the front of each note? If we are not keeping it, what is the ui to show?

Comment (Arighna88) [reply app/src/main/kotlin/com/twain/interprep/presentation/ui/components/note/NoteCard.kt:84]: We will keep the index.

Comment (sev3rus) [reply app/src/main/kotlin/com/twain/interprep/presentation/ui/components/note/NoteCard.kt:84]: btw I think the index should start from 1?

Comment (Arighna88) [reply app/src/main/kotlin/com/twain/interprep/presentation/ui/components/note/NoteCard.kt:84]: I was about to write the same thing. Index should start from 1

File: app/src/main/kotlin/com/twain/interprep/presentation/ui/components/note/InterviewDetailForNote.kt (hunk 1 above)
Comment (Arighna88) [reply app/src/main/kotlin/com/twain/interprep/presentation/ui/components/note/InterviewDetailForNote.kt:74]: It should not be ideally a fixed height. All the Text() should be single line with ending ellipsis.

**Review by Arighna88 — APPROVED** [review]
Remove fixed height - We will take it up in next PR.

**Pull request merged**